esp-wifi/src/lib.rs

```diff
@@ -89,20 +89,35 @@ pub fn current_millis() -> u64 {
     get_systimer_count() / (TICKS_PER_SECOND / 1000)
 }
 
+#[allow(unused)]
 #[cfg(all(not(coex), not(feature = "big-heap")))]
-const HEAP_SIZE: usize = 64 * 1024;
+const DEFAULT_HEAP_SIZE: usize = 64 * 1024;
 
+#[allow(unused)]
 #[cfg(all(coex, not(feature = "big-heap")))]
-const HEAP_SIZE: usize = 64 * 1024;
+const DEFAULT_HEAP_SIZE: usize = 64 * 1024;
```

Comment:
`coex` check seems redundant on this one and above (one is `coex & big heap`, other is `!coex & big_heap`, both with the same value), in case you want to simplify :)

Reply:
true - most probably I will remove the big-heap feature and only have two defaults which somehow work everywhere ... we don't need to find the one best value (which doesn't exist for every one and every situation) anymore since it's now configurable